app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -510,18 +515,19 @@ class SamplesHeatmapView extends React.Component {
     thresholdFilters: this.props.thresholdFilters,
     // Client side options
     scales: SCALE_OPTIONS,
+    sortSamplesOptions: SORT_SAMPLES_OPTIONS,
     taxonsPerSample: TAXONS_PER_SAMPLE_RANGE,
     specificityOptions: SPECIFICITY_OPTIONS,
   });
 
   handleSelectedOptionsChange = newOptions => {
-    const refetchData = !isEqual(keys(newOptions), ["dataScaleIdx"]);
-
+    const excluding = ["dataScaleIdx", "sortSamples"];
+    const refetchData = difference(keys(newOptions), excluding);
```

Comment:
Could we do `shouldRefetchData = difference(keys(newOptions), excluding).length === 0`? Makes the naming clearer.

Reply:
👍